### ISSUE TITLE
What is TR pin?

### ISSUE THREAD

**Issue by rockerrock137** [meatpiHQ/wican-fw#29]
Is TR Termination Resistor? If so, can please document how to use?

**Comment by meatpiHQ** [comment]
@rockerrock137 Yes TR is the termination resistor. Connect TR to CANH and it will terminate the bus. The other end of the resistor is connected the CANL internally.
